### PR TITLE
Add Google geocoding example

### PR DESCRIPTION
**Pull request by kuldarkalvik**
Hey guys, I had to create a small function for fetching lat/lng from an inputted address. I thought I'd share the small function. I'm quite green at GraphCool and quite.. yellow? at Javascript, so all suggestions how to improve the function are more than welcome. 🙂

### CODE REVIEW

File: google-geocoding/google-geocoding.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+require('isomorphic-fetch')
+
+module.exports = function (event) {
+  var baseUrl = 'https://maps.googleapis.com/maps/api/geocode/json'
+  var apiKey = '__GOOGLE_API_KEY__'
+
+  // Here we're merging contents of three inputs – country, city and street.
+  // You could also have this be just one freestyle address input
+  var address = encodeURI([event.data.street, event.data.city, event.data.country].join('+').replace(/\s/g, '+'))
```
Comment (marktani) [inline google-geocoding/google-geocoding.js]: can you explain the RegEx? Not my strength 🙈

Comment (marktani) [inline google-geocoding/google-geocoding.js]: are you replacing spaces with `+` here?

Comment (kuldarkalvik) [inline google-geocoding/google-geocoding.js]: Not strong on Regex either, but yeah, swapping spaces for plusses here.

Comment (kuldarkalvik) [inline google-geocoding/google-geocoding.js]: Not strong on Regex either, but yeah, swapping spaces for plusses here.

**Comment by marktani** [comment]
Thanks so much for putting this together @kuldarkalvik and congratulations to your first contribution 🎉

**Comment by kuldarkalvik** [comment]
Awesome, happy to contribute in a tiny way! Thanks for the code improvements as well. 👍

**Pull request merged**